robottelo/common/constants.py

```diff
@@ -56,6 +61,7 @@
 #       filter's common_locator['filter']
 FILTER = {'arch_os': "architecture_operatingsystem",
```

Comment:
I am sorry for my inattentiveness. Probably, it was 'Friday evening syndrome' :) Will use provided example